### PR TITLE
Step3

### PR DESCRIPTION
**Pull request by carloscadena**

### CODE REVIEW

File: pyramid_learning_journal/pyramid_learning_journal/scripts/initializedb.py [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
 from ..models import (
     get_engine,
     get_session_factory,
     get_tm_session,
     )
-from ..models import MyModel
+from pyramid_learning_journal.data.data import JOURNALS
+from ..models import Journal
+from datetime import datetime
+from faker import Faker
```
Comment (pasaunders) [inline pyramid_learning_journal/pyramid_learning_journal/scripts/initializedb.py:19]: This isn't in your setup.py

File: pyramid_learning_journal/pyramid_learning_journal/views/default.py [hunk 2]
```diff
@@ -1,43 +1,29 @@
...
+    session = request.dbsession
+    entry = session.query(Journal).get(the_id)
+    if not entry:
         raise HTTPNotFound
 
     return {
         'page': 'Journal Entry',
-        'journal': journal
+        'entry': entry
     }
```
Comment (pasaunders) [inline pyramid_learning_journal/pyramid_learning_journal/views/default.py:29]: You should also have add and edit views, even if they just return an empty dict.

File: pyramid_learning_journal/pyramid_learning_journal/tests.py [hunk 3]
```diff
@@ -1,36 +1,150 @@
...
+    from pyramid_learning_journal.views.default import home_view
+    response = home_view(dummy_request)
+    assert isinstance(response, dict)
+
 
+def test_home_view_returns_empty_when_database_empty(dummy_request):
+    """Home view returns nothing when there is no data."""
+    from pyramid_learning_journal.views.default import home_view
+    response = home_view(dummy_request)
+    assert len(response['journals']) == 0
```
Comment (pasaunders) [inline pyramid_learning_journal/pyramid_learning_journal/tests.py:136]: Good job getting passing tests. Now try to expand on them. I'd like to see tests for edge cases and expected failures. Test your error handling.

**Review by pasaunders — COMMENTED** [review]
This is a good start, just test more thoroughly and fill out the assignment to fit spec. I'm glad to see the heroku deployment.
